### PR TITLE
[ MB-7952, MB-11694 ] Sign out should sign users out of MM and login.gov

### PR DESCRIPTION
**Pull request by rogeruiz**
## Summary

[=> View this Slack thread for more information 🔒 ](https://ustcdp3.slack.com/archives/CP6PTUPQF/p1645569111796149)

Basically, we are testing that when a user is logged out that they are taken to a URL / Page that explicitly says they have been logged out of the system. And that "being logged out" would mean they're logged out of both MM and login.gov sessions.

## Setup to Run Your Code

This code is best tested in with the Review App Bot or on Experimental.

### Additional steps

1. Login via login.gov
1. Logout using the Logout link in the header of the app
1. Try to log in again
1. You should see a login.gov login screen
    1. You **should not** see a login.gov screen that shows you're already logged in.

## Verification Steps for Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Request review from a member of a different team.
- [ ] Have the Jira acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Frontend

- [ ] User facing changes have been reviewed by design.
- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
- [ ] There are no new console errors in the browser devtools
- [ ] There are no new console errors in the test output
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.

### CODE REVIEW

**Comment by rogeruiz** [comment]
Converting this to Draft as I'm deploying it to Experimental and don't want it to merge accidentally.

File: .circleci/config.yml [hunk 1]
```diff
@@ -59,12 +59,12 @@ references:
...
   # client tests, or `placeholder_branch_name` if you do want to run
   # them
-  client-ignore-branch: &client-ignore-branch placeholder_branch_name
+  client-ignore-branch: &client-ignore-branch client-ignore-branch
 
   # set server-ignore-branch to the branch if you want to IGNORE
   # server tests, or `placeholder_branch_name` if you do want to run
   # them
-  server-ignore-branch: &server-ignore-branch placeholder_branch_name
+  server-ignore-branch: &server-ignore-branch server-ignore-branch
```
Comment (rogeruiz) [inline .circleci/config.yml:68]: Revert these changes before taking this PR out of the **Draft** state.

Comment (jacquelineIO) [reply .circleci/config.yml:68]: @rogeruiz what are these changes for? (I do see that they are only temp)

**Review by LeDeep — APPROVED** [review]
looks good. Tested on experimental and tried logging out, and was asked to login in again.
